docs/DeveloperGuide.md

```diff
@@ -236,42 +236,47 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* Tertiary students
+* Has a need to track everyday expenses/income
+* Wishes to cultivate good saving habits
+* Prefers desktop apps
+* Prefers typing to mouse interactions
 
+**Value proposition**: manage finances and cultivate good financial habits (such as saving) efficiently by typing in CLI commands
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​        | I want to …​                 | So that I can…​                                                     |
```

Comment:
Should the column be made larger so that the text fits in a single line?

![image](https://user-images.githubusercontent.com/5585517/93794755-16a78900-fc6b-11ea-9fcc-358276341966.png)